lib/mocha/inspect.rb

```diff
@@ -18,8 +18,9 @@ def mocha_inspect(wrapped = true)
     end
 
     module HashMethods
-      def mocha_inspect(wrapped = true)
+      def mocha_inspect(placeholder = nil)
```

Comment:
I don't think we need this arg, will clean this up if we are good with the approach.

Reply:
Hmm. As far as I can see (in contrast to the same mechanism on `ArrayMethods#mocha_inspect`) it's not obvious that `HashMethods#mocha_inspect` has _ever_ been used with `wrapped` set to `false` other than from [its unit test](https://github.com/freerange/mocha/blob/af2194c4f4465200fd07fd078f2dbefd9f712360/test/unit/hash_inspect_test.rb#L12). If that's correct, I'd be inclined to remove this in a separate commit in the `main` branch. Let me know and I can make that happen.

Reply:
Fixing as a separate commit makes sense to me! Would I leave it to you to copy changes from this spike onto https://github.com/freerange/mocha/pull/562 (pending the discussion in [this comment](https://github.com/wasabigeek/mocha/pull/2#issuecomment-1272525903))?

Reply:
> Would I leave it to you to clean up / copy changes from this spike onto [freerange#562](https://github.com/freerange/mocha/pull/562) (pending the discussion in [this comment](https://github.com/wasabigeek/mocha/pull/2#issuecomment-1272525903))?

Yes, that's fine - thank you!

Reply:
Addressed in https://github.com/freerange/mocha/commit/4f59e27f06dad87d86163ad680db91b0accf0984.